primitives/src/targeting/eval.rs

```diff
@@ -12,12 +12,46 @@ pub type Map = serde_json::value::Map<String, SerdeValue>;
 
 use super::{Input, Output};
 
+#[cfg(test)]
+#[path = "eval_test.rs"]
+mod test;
+
 #[derive(Debug, Eq, PartialEq)]
 pub enum Error {
     TypeError,
     UnknownVariable,
 }
 
+fn get_deposit_asset_divisor(address: &String) -> Result<BigNum, Error> {
+    match address.as_str() {
```

Comment:
Could you check what should happen if the address is not found in the list? Maybe a `None` for the price would be better?!

We should probably add `USDC` as well. Maybe `DIA`, `USDT` & `USDC` is sufficient list?
 
@Ivshti maybe you could shine a light on the two questions?